prow/plugins/label/label.go

```diff
@@ -190,6 +192,12 @@ func handle(gc githubClient, log *logrus.Entry, additionalLabels []string, e *gi
 		return gc.CreateComment(org, repo, e.Number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, e.User.Login, msg))
 	}
 
+	if len(noSuchLabelsInRepo) > 0 {
+		log.Infof("Labels missing in repo: %v", noSuchLabelsInRepo)
+		msg := fmt.Sprintf("The label(s) `%s` cannot be applied, because the repository doesn't have them", strings.Join(noSuchLabelsInRepo, ", "))
+		return gc.CreateComment(org, repo, e.Number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, e.User.Login, msg))
```

Comment:
Wouldn't it be better to still create the valid labels instead of returning here?

That would also make the change more backwards-compatible.

/ok-to-test

Reply:
The labels are still created in the the for loop.